utils/ansible/ftest/templates/daos-launch_nlt.sh.j2

```diff
@@ -0,0 +1,216 @@
+#!/bin/bash
```

Comment:
This looks nice but it would be good to have some description of what it does that the other numerous scripts we have don't, also is it referenced anywhere else to tell people when it would be useful? Readme or header documentation needed.